index.html

```diff
@@ -1,13 +1,65 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <!-- Don't forget to change the title and connect the CSS file -->
-  <title>Project Library</title>
-</head>
-<body>
-  <h1>Project Library</h1>
-  <!-- Don't forget to connect the JavaScript file -->
-</body>
-</html>
\ No newline at end of file
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Book Library</title>
+    <link rel="stylesheet" href="./style.css" />
+  </head>
+  <body>
+    <h1>Project Library</h1>
+
+    <!--FILTER & SORTING SYSTEM-->
+    <div class="sort-buttons">
+      <!--filter by genre-->
+      <div class="label-dropdown">
+        <label for="filter-genre">Filter by genre:</label>
+        <select class="dropdown-genre" id="filter-genre">
+          <option value="all">All</option>
+          <option value="adventure">Adventure</option>
+          <option value="dystopian">Dystopian</option>
+          <option value="fantasy">Fantasy</option>
+          <option value="fiction">Fiction</option>
+          <option value="horror">Horror</option>
+          <option value="mystery">Mystery</option>
+          <option value="science fiction">Science Fiction</option>
+        </select>
+      </div>
+
+      <!--filter by year-->
+      <div class="label-dropdown">
+        <label for="filter-year" filter-year>Filter by year:</label>
+        <select class="dropdown-genre" id="filter-year">
+          <option value="all">All Years</option>
+          <option value="19th">19th Century (1801-1900)</option>
+          <option value="20th">20th Century (1901-2000)</option>
+          <option value="post20th">After 20th Century</option>
+        </select>
+      </div>
+
+      <!--button for random book-->
+      <div class="label-dropdown">
+        <button id="random-book">Shuffle books</button>
```

Comment:
little note: not sure this text explains the function of your button. Select a random book and shuffle the books are two slightly different things. Text, info and instructions in your app need to be accurate for the best user experience 👍